nrpe_exporter.go

```diff
@@ -42,9 +43,9 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- prometheus.NewDesc("dummy", "dummy", nil, nil)
 }
 
-func collectCommandMetrics(cmd string, conn net.Conn, logger log.Logger) (CommandResult, error) {
+func collectCommandMetrics(cmd string, args string, conn net.Conn, logger log.Logger) (CommandResult, error) {
 	// Parse and issue given command
-	command := nrpe.NewCommand(cmd)
+	command := nrpe.NewCommand(cmd, args)
```

Comment:
This doesn't seem right, this only allows passing one argument.

Reply:
you can pass multiple arguments separated by `!` (the nrpe way of separating args) like in my documented example
`args: [1:1!1:1!/opt/prometheus/alertmanager]`

Reply:
That doesn't seem the most general or secure way of doing things, why not a parameter per arg?